TheaterSchedule.DAL/Interfaces/IRepository.cs

```diff
@@ -1,10 +1,28 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq.Expressions;
+using TheaterSchedule.DAL.Entities;
 
 namespace TheaterSchedule.DAL.Interfaces
 {
     public interface IRepository<T> where T : class
+    {
+        IEnumerable<T> GetAll();
```

Comment:
start implementation only with methods that you really need
Why this is important:
If we will have code coverage then you will be forced to cover by tests all these methods
Also classes will be clean and without code that is not used